android/pytorch_android_torchvision/CMakeLists.txt

```diff
@@ -1,21 +1,19 @@
 cmake_minimum_required(VERSION 3.5)
 project(pytorch_vision_jni CXX)
-set(CMAKE_CXX_STANDARD 17 CACHE STRING "The C++ standard whose features are requested to build this target.")
+set(CMAKE_CXX_STANDARD
+    17
+    CACHE STRING
+          "The C++ standard whose features are requested to build this target.")
```

Comment:
this seems kind of ugly but it's a cost I'm willing to pay to have more than zero auto-formatting, personally